src/shmem_env_defs.h

```diff
@@ -109,6 +109,11 @@ SHMEM_INTERNAL_ENV_DEF(OFI_STX_DISABLE_PRIVATE, bool, false, SHMEM_INTERNAL_ENV_
                        "Disallow private contexts from having exclusive STX access")
 #endif
 
+#ifdef USE_UCX
+SHMEM_INTERNAL_ENV_DEF(PROGRESS_INTERVAL, long, 1000, SHMEM_INTERNAL_ENV_CAT_TRANSPORT,
+                       "Polling interval for progress thread in microseconds (0 to disable)")
+#endif
```

Comment:
@jdinan Does UCX have any similar parameter for users? I am thinking, for PSM2, we did not introduce any such parameter in the SHMEM layer. And this will also cause additional threading in the SHMEM, right? Also, will this parameter need to be tuned? Should we expect different behavior for different MLX NICs?

Reply:
No, UCX does not spawn a thread to generate automatic progress. As with all progress threads, there is no perfect value to put here and performance will vary based on system, application, phase of the moon, etc. For benchmarking runs (and most applications) the progress thread can be disabled, but it is needed to be fully compliant with the OpenSHMEM spec.